samples/precompiled/Monitor.cs

```diff
@@ -11,7 +11,32 @@
 using Newtonsoft.Json.Linq;
 using Twilio;
 
-// To run this sample, you'll need a Weather Underground API key. Sign-up and documentation here: https://www.wunderground.com/weather/api/d/docs
+/* This sample demonstrates the Monitor workflow. In this pattern, the orchestrator function is
```

Comment:
I really like these detailed overview comments. We should definitely start doing more of this in our other samples.